### PR TITLE
fix notifications in cms

### PR DESCRIPTION
**Pull request by benghaziboy**
@kand @mparent61 

think some of the cms stuff broke notifications, fixes here.

### CODE REVIEW

File: app/components/campaigns/campaigns-list/campaigns-list-page.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <cms-notify-container class="cms-notify-container"></cms-notify-container>
 
   <list-page
-      cms-edit-page-url="'/cms/app/campaigns/edit/' + item.id"
+      cms-edit-page-url="'/cms/app/notifications/edit/' + item.id"
```
Comment (mparent61) [inline app/components/campaigns/campaigns-list/campaigns-list-page.html]: ??

Comment (benghaziboy) [inline app/components/campaigns/campaigns-list/campaigns-list-page.html]: oh god split screen; ignore me

**Comment by mparent61** [comment]
LGTM

**Comment by kand** [comment]
:+1:

**Comment by kand** [comment]
:+1:

**Comment by kand** [comment]
:+1:

**Comment by kand** [comment]
:+1:

**Comment by kand** [comment]
:+1:

**Comment by kand** [comment]
:+1:

**Pull request merged**